### ISSUE TITLE
Custom videos do not work

### ISSUE THREAD

**Issue by ilovecats516** [JohnCoates/Aerial#906]
#### General troubleshooting tips

Before logging an issue please check that:
- [x] You have the latest version installed (There may be a beta version that fixes your issue), see here for the latest releases and bug fixes: https://github.com/JohnCoates/Aerial/releases
- [x] Your issue isn't already mentioned in our [issues](https://github.com/JohnCoates/Aerial/issues). You may find a workaround there or a similar request already made.
- [x] Your problem isn't mentioned in the [troubleshooting section](https://github.com/JohnCoates/Aerial#troubleshooting).

If none of this fixes your issue, tell us about the problem you are experiencing or the feature you'd like to request.

#### Required information
In order to help us sort your issue, we ask that you provide the following information:
- [ ] Mac model: iMac 21.5 inch, Late 2012
- [ ] macOS version: MacOS Catalina
- [ ] Monitor setup: 1 monitor

If appropriate, please enable `Debug mode` and `Log to disk` in `Advanced` tab and replicate your bug, then attach the `AerialLog.txt` file (You can access this file through the Advanced tab).

#### Description of issue / Feature request

In System Preferences, my custom videos run fine in the preview box in Aerial settings, but will not run when the screensaver launches.

AerialLog.txt : 
[AerialLog.txt](https://github.com/JohnCoates/Aerial/files/4093794/AerialLog.txt)

**Comment by glouel** [comment]
Hi @ilovecats516 

I can confirm the bug in Catalina, I'll have a look and get back to you.

**Comment by glouel** [comment]
Ok, I'm not 100% sure why but we are denied access to the files. 

Here's what I got here : 
`par défaut	16:02:14.022536+0100	kernel	Sandbox: 2 duplicate reports for legacyScreenSave deny(1) file-read-data /Users/glouel/Documents/Custom Videos/Mon film 1.mp4`

My assumption so far is we're denied access because it's in the user folder, in a place where user usually store their stuff. It was my understanding that accessing a library's user folder was ok though in Catalina, unless that changed, so I guess user accessible folders are extra protected (I used Documents, you used Desktop, but we had the same result). 

Two workarounds I found so far: 
- put the files in the sandboxed container : `~/Library/Containers/com.apple.ScreenSaver.Engine.legacyScreenSaver/Data/Library/Application Support/Aerial`
- put the files in the shared user folder (probably the easiest !) : `/Users/Shared`

I'd recommend you try the second one and let me know. Sorry about that but after all those months I'm still not 100% clear of all the intricacies of how Catalina restricts screensavers...
